services/processor/native/repository/_Elections/fix_rewards_and_stakes.go

```diff
@@ -0,0 +1,154 @@
+	{"fa76362c7937b02018abd99c04333d3b62ff7f3d", 624},   // instead of 1092
+	{"9e2d2a783274c81337fec2e2897fee5ec614cca0", 539},   // instead of 490
+	{"e933cedb1d7772ab3ca6ca8c173cc6b6ad61bb5f", 140},   // instead of 217
+	{"0a91147313b82fdb20c853646e829b8d32a9fdb8", 1319},  // instead of 1879
+	{"f07df0a74967c53611f2bb619eec462527ddb5d3", 297},   // instead of 270
+	{"fddb0aa468c70be6440c46cfbdf099d2135e25a7", 2285},  // instead of 2287
+	{"960ac2fd49b2088cfbb385fed12adb4d78429928", 6772},  // instead of 11800
+	{"f061c6acc642676914aa7139abc89fbf97a06112", 20834}, // instead of 32198
+	{"aa1a4565f2d875995d85e8d52e7193c666e20c04", 647},   // instead of 961
+	{"8fd0a7b70aa896cf85b3034385f98af1d927d442", 4082},  // instead of 0
+}
+
+var guardiansFixDoubleDelegationReward = []struct {
+	address string
+	reward  int
+}{
+	{"f7ae622C77D0580f02Bcb2f92380d61e3F6e466c", 1399658}, // instead of 1403641
+	{"63AEf7616882F488BCa97361d1c24F05B4657ae5", 1250192}, // instead of 1250217
+	{"C82eC0F3c834d337c51c9375a1C0A65Ce7aaDaec", 865579},  // instead of 855152
+	{"9afc8EF233e2793B2b90Ca5d70cA2e7098013142", 522112},  // instead of 522649
+	{"8287928a809346dF4Cd53A096025a1136F7C4fF5", 340047},  // instead of 340157
+	{"F058cCFB2324310C33E8FD9a1ddA8E99C8bEdA59", 527823},  // instead of 528491
+	{"a3cBDD66267DAaA4B51Af6CD894c92054bb2F2c7", 21201},   // instead of 47243
+	{"cB6172196BbCf5b4cf9949D7f2e4Ee802EF2b81D", 94199},   // instead of 85826
+	{"1763F3DA9380E2df7FfDE1dC245801BB14F80669", 21887},   // instead of 15898
+}
+
+func _fixRewardsDrift7928900() {
+	key := []byte("_fixRewards7928900_")
+	if state.ReadUint32(key) == 0 {
+
+		for _, participant := range participantsFixDoubleDelegationReward {
+			bytes, err := hex.DecodeString(participant.address)
+			if err != nil {
+				panic(fmt.Errorf("cannot parse %s , err %s", participant.address, err))
+			}
+			wrongValue := state.ReadUint64(_formatCumulativeParticipationReward(bytes))
+			state.WriteUint64(_formatCumulativeParticipationReward(bytes), uint64(participant.reward))
+			newValue := state.ReadUint64(_formatCumulativeParticipationReward(bytes))
+			fmt.Printf("elections fix rewards: Participant %s reward changed from %d to %d\n", participant.address, wrongValue, newValue)
+		}
+
+		for _, guardian := range guardiansFixDoubleDelegationReward {
+			bytes, err := hex.DecodeString(guardian.address)
+			if err != nil {
+				panic(fmt.Errorf("cannot parse %s , err %s", guardian.address, err))
+			}
+			wrongValue := state.ReadUint64(_formatCumulativeGuardianExcellenceReward(bytes))
+			state.WriteUint64(_formatCumulativeGuardianExcellenceReward(bytes), uint64(guardian.reward))
+			newValue := state.ReadUint64(_formatCumulativeGuardianExcellenceReward(bytes))
+			fmt.Printf("elections fix rewards: Guardian %s reward changed from %d to %d\n", guardian.address, wrongValue, newValue)
+		}
+
+		state.WriteUint32(key, 1)
+	} else {
+		panic(fmt.Sprintf("cannot fix rewards anymore"))
```

Comment:
I wouldn't panic here. Just return error. (same for the next method)

Reply:
this one i must ignore as currently when the practice for error is to panic. Otherwise there will be a committed successfully return message (but nothing will have changed).